inst/include/intermediate_data.hpp

```diff
@@ -89,6 +89,9 @@ const int MALE = 0;
 const int FEMALE = 1;
 const int ART0MOS = 0;
 
+const int PROJPERIOD_MIDYEAR = 0;   // mid-year projection period 
+const int PROJPERIOD_CALENDAR = 1;  // calendar-year projection (Spectrum 6.2 update; December 2022)
```

Comment:
should the calendar year be zero? I conceptualize 0 as the default, and seems that calendar year will be default going forward

Reply:
Thanks, this is good suggestion. (They were defined on this based on the sequencing of implementation in EPP-ASM)

Reply:
Done in 952c2a8. Also have replaced hardcoding the value to calling the macro